release-notes.md

```diff
@@ -1,5 +1,17 @@
 # Release notes
 
+## Sep 9, 2019
+
+The following updates were made in this release:
+
+**Android Griffon SDK 1.0.0**
+
+* New Artifact split out from the Android Griffon Bridge as of version 1.0.3.
```

Comment:
Please reword the 1st bullet:

"Starting in version 1.0.3, there is a new Artifact split from the Android Griffon Bridge."